addons/website_sale/models/product.py

```diff
@@ -24,7 +24,7 @@ def _default_website(self):
     selectable = fields.Boolean(help="Allow the end user to choose this price list")
 
     def clear_cache(self):
-        # website._get_pl() is cached to avoid to recompute at each request the
+        # website._get_pl_partner_order() is cached to avoid to recompute at each request the
```

Comment:
merge this commit with another one test commit, since it doesn't fix anything really except maybe a test

Reply:
As discussed, I will let this commit alone on top of the PR to avoid rebase nightmares.